api_fhir_r4/converters/coverageEligibilityRequestConverter.py

```diff
@@ -17,10 +16,10 @@ def to_fhir_obj(cls, coverage_eligibility_response):
 
     @classmethod
     def to_imis_obj(cls, fhir_coverage_eligibility_request, audit_user_id):
-        uuid = cls.build_imis_uuid(fhir_coverage_eligibility_request)
+        chf_id = cls.build_imis_uuid(fhir_coverage_eligibility_request)
```

Comment:
Variable name doesn't match with method name.

Reply:
ok. It was changed in that commit https://github.com/openimis/openimis-be-api_fhir_r4_py/pull/17/commits/c93cd4aa34259e978aba46fb5cf5ac8ea8e952cc